### PR TITLE
RNA data loading and filtering

### PR DESCRIPTION
**Pull request by jaceybronte**
This PR loads the RNAseq dataset in and filters the top 2000 genes into training, testing, and validation

### CODE REVIEW

File: 6.RNAseq/scripts/2.filter_data.py [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+
+
+import pandas as pd
+import pathlib
+import sys
+from sklearn.preprocessing import StandardScaler
+
+script_directory = pathlib.Path("../utils/").resolve()
+sys.path.insert(0, str(script_directory))
+from data_loader import load_train_test_data
```
Comment (axiomcura) [inline 6.RNAseq/scripts/2.filter_data.py:14]: Consider following pep8 rules in order to make importing less stressful for python. [docs](https://peps.python.org/pep-0008/#imports)

File: 6.RNAseq/scripts/2.filter_data.py [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+    filtered_rna_split = aligned_rna_data[['SampleID'] + list(top_genes)]
+
+    # Save the filtered split
+    if output_dir:
+        output_path = pathlib.Path(output_dir) / f"RNASeq_{split_name}_filtered.parquet"
+        filtered_rna_split.to_parquet(output_path, index=False)
+        print(f"Saved {split_name} split to {output_path}")
+        print(filtered_rna_split.shape)
+
+    return filtered_rna_split
```
Comment (axiomcura) [inline 6.RNAseq/scripts/2.filter_data.py]: Consider placing functions at the top of the notebook to improve readability

File: 6.RNAseq/scripts/2.filter_data.py [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+
+    # Save z-scored data if output directory is specified
+    if output_dir:
+        output_path = pathlib.Path(output_dir) / f"RNASeq_{split_name}_zscored.parquet"
+        zscored_data.to_parquet(output_path, index=False)
+        print(f"Saved {split_name} split to {output_path}")
+
+    print(zscored_data.head())
+    print(zscored_data.shape)
+    return zscored_data
```
Comment (axiomcura) [inline 6.RNAseq/scripts/2.filter_data.py:90]: Same comment above

**Review by axiomcura — APPROVED** [review]
LGTM! nice work c: Left some comments

File: 6.RNAseq/scripts/2.filter_data.py (hunk 1 above)
Comment (jaceybronte) [reply 6.RNAseq/scripts/2.filter_data.py:14]: This is helpful thanks!

File: 6.RNAseq/scripts/2.filter_data.py (hunk 2 above)
Comment (jaceybronte) [reply 6.RNAseq/scripts/2.filter_data.py]: done

File: 6.RNAseq/scripts/2.filter_data.py (hunk 3 above)
Comment (jaceybronte) [reply 6.RNAseq/scripts/2.filter_data.py:90]: done

**Pull request merged**